infor-design/enterprise-wc
# 533 - Add menu button features

**Explain the details for making this change. What existing problem does the pull request solve?**
Adds remaining features to ids-menu-button. 
Toggles `is-active` class on menu button when popup is open/closed.
Adds get/set `value` on menu button to get/set selected items in ids-popup-menu

**Related github/jira issue (required)**:
Closes #533 

**Steps necessary to review your pull request (required)**:
1. Checkout branch, `npm install`, `npm run start`
2. Go to http://localhost:4300/ids-menu-button 

**Testing `is-active` state**
3. Click menu button to open popupmenu
4. Check that focus state remains on menu button and `is-active` class applied to ids-menu-button > button 
5. Close popupmenu and check that `is-active` class is removed 

**Testing `value` setting**
6. Open dev tools, test `value` setting 
7. Ex. `$('#icon-button').value = ['1', '2'];`
8. Ex. `$('#menu-button').value = 'small';`

**Included in this Pull Request**:
